modules/ROOT/pages/db/db-connector-sp.adoc

```diff
@@ -1,11 +1,11 @@
-= Execute Stored Procedures - Mule 4
+= Example: Execute Stored Procedures - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Invoke stored procedures that combine, input, output, and input-output parameters. 
+Create a stored procedures that allows the input, output, and input-output of payload parameters.
```

Comment:
This doesn't create stored procedures, invokes stored procedures.
Doesn't exist such thing as "payload parameters"